x-pack/plugins/siem/public/components/flyout/header/index.tsx

```diff
@@ -58,9 +60,9 @@ interface DispatchProps {
     }
   ) => void;
   createTimeline?: ({ id, show }: { id: string; show?: boolean }) => void;
+  toggleLock?: ({ linkToId }: { linkToId: inputsModel.InputsModelId }) => void;
```

Comment:
All of these that are TypeScript FSA could be typed as `ActionCreator` like this example:

```ts
  createTimeline: ActionCreator<{
    id: string;
    show?: boolean;
  }>;
```

That's usually how people type these rather than duplicating the signatures.

Reply:
I know, I was just trying to keep it the way it was, I will refactor ;)